ariel/generate_reports.py

```diff
@@ -52,8 +52,19 @@ def get_units(instancetype):
     family_value = instances['instancetype'].apply(lambda x: x.split('.')[0])
     instances.insert(family_column, 'instancetypefamily', family_value)
 
-    # Amazon still hasn't fixed g4dn, so we need to filter out instance types that we don't have size data about.
+    # Amazon still hasn't fixed g4dn, so we need to filter out instance types and RIs that we don't have size data about.
     instances = instances[instances.instancetype.isin(pricing['units'].keys())].reset_index(drop=True)
+    ris = ris[ris.instancetype.isin(pricing['units'].keys())].reset_index(drop=True)
+
+    # Filter out instances and RIs we're not interested in
+    skip_accounts = utils.get_config_value(config, 'RI_PURCHASES', 'SKIP_ACCOUNTS', '').split(' ')
+    instances = instances[~instances.usageaccountid.isin(skip_accounts)].reset_index(drop=True)
+    ris = ris[~ris.accountid.isin(skip_accounts)].reset_index(drop=True)
+
+    include_accounts = utils.get_config_value(config, 'RI_PURCHASES', 'INCLUDE_ACCOUNTS', '').split(' ')
```

Comment:
Is this parameter required?  If it is not set, what is the default (just the account)?  Is Ariel organization aware so it can just check all accounts in the organization without having to pass in a list?

Reply:
It is not required.  If you do not specify include accounts, it includes everything.  This is a way to filter to only a subset of accounts.  Much like SKIP_ACCOUNTS lets you filter out a subset of accounts.

Reply:
Let me get this added to the example config.